### PR TITLE
Redesigned to do

### PR DESCRIPTION
**Pull request by akssinhaamit**
# What and why?
Redesign survey list to do page as per the new story
# How to test?
forntstage smoke test
# Trello
https://trello.com/c/QY8jiD6E/515-user-self-serve-to-do-tab-page-design-alterations

### CODE REVIEW

File: frontstage/controllers/collection_exercise_controller.py [hunk 1]
```diff
@@ -88,8 +89,83 @@ def convert_events_to_new_format(events):
...
+        return 'Due in a month'
+    if two_months <= event_day < three_months:
+        return 'Due in 2 months'
+    if three_months <= event_day < four_months:
+        return 'Due in 3 months'
+    if four_months <= event_day:
+        return 'Due in over 3 months'
+
+
+def return_date_time(timedelta_now: datetime):
```
Comment (insacuri) [inline frontstage/controllers/collection_exercise_controller.py:165]: Pretty minor, but the signature can also include the return value.
```python
def return_date_time(timedelta_now: datetime) -> datetime:
```

**Review by insacuri — CHANGES_REQUESTED** [review]
A few minor changes but it looks good

File: frontstage/controllers/collection_exercise_controller.py [hunk 2]
```diff
@@ -88,8 +89,83 @@ def convert_events_to_new_format(events):
...
+
+    :param: date_format_required: output format in which date should be returned
+    :param: date_to_be_formatted: the datetime object which needs ordinal date
+    :return: formatted date
+    """
+    return date_to_be_formatted. \
+        strftime(date_format_required).replace('{S}', str(date_to_be_formatted.day) + suffix(date_to_be_formatted.day))
+
+
+def due_date_convertor(date: datetime):
```
[the remark this answers is not in the document]
Comment (akssinhaamit) [reply frontstage/controllers/collection_exercise_controller.py]: 👍

File: frontstage/controllers/collection_exercise_controller.py (hunk 1 above)
Comment (akssinhaamit) [reply frontstage/controllers/collection_exercise_controller.py]: 👍

**Review by insacuri — APPROVED** [review]

**Pull request merged**